test/test_rospy/test/rostest/test_rospy_client_online.py

```diff
@@ -68,141 +68,176 @@ def __init__(self, *args):
         rospy.init_node('test_rospy_online')
 
     def test_log(self):
-        # we can't do anything fancy here like capture stdout as rospy
-        # grabs the streams before we do. Instead, just make sure the
-        # routines don't crash.
-        
-        real_stdout = sys.stdout
-        real_stderr = sys.stderr
 
+        import logging
+        import rosgraph.roslogging
```

Comment:
I actually missed these...
I also noticed the test_wait_for_service has a comment "lazy-import for coverage"... Is this something I should be concerned about ? I don't know how I can check coverage in this repo... any doc you could point me to, describing this ?